scripts/publish.sh

```diff
@@ -23,12 +22,6 @@ crates=(
 )
 
 for crate in "${crates[@]}"; do
-    crate_manifest=$(cargo metadata --no-deps --format-version 1 |
-    jq -r --arg crate "$crate" '.packages[] |
-    select(.name == $crate) |
-    .manifest_path')
-
     # Publish to crates.io
-    cargo publish --manifest-path "$crate_manifest" --token "$CRATES_IO_TOKEN"
-    cargo clean --manifest-path "$crate_manifest"
+    cargo publish --package "$crate" --token "$CRATES_IO_TOKEN" || echo "failed"
```

Comment:
In this case, we won't have any alerts if the publish job fails, no?

Reply:
Will remove the `OR` after re-publish. 

A side note, there were no alerts before : ) It started failing because I added `set -euxo pipefail` to the script in https://github.com/ChainSafe/fil-actor-states/pull/307